### DISCUSSION TITLE
Import for multiple users

### DISCUSSION THREAD

**Discussion by RazorII** [Taxel/PlexTraktSync#1557] [Q&A] [answered]
I have a few Plex users in my family & I'd like to import the trakt watched status for all of them. It's need the Plex login info for each user & their trakt login and do it once for each of the users. That's okay if it's a one time deal but if I want to keep Trakt in sync with Plex & vice versa, how can do that for multiple users?

**Comment by glensc** [comment, marked as answer]
there are multiple discussions and with solutions. use the search:
- https://github.com/Taxel/PlexTraktSync/discussions?discussions_q=multiple+users

For example:
- https://github.com/Taxel/PlexTraktSync/discussions/601
